### PR TITLE
TS || EditPostsTicket8

### PR DESCRIPTION
**Pull request by TabathaSlater**
# Description

**This code addresses ticket 8 (Edit Posts). 

https://trello.com/b/teDb9crL/the-candy-corns

Given an author is viewing a Post that they have written
When they select the delete option
Then they should be presented to confirm the deletion

Given the author wishes to confirm the delete
When they the select the option to confirm
Then the Post should be removed from the system
And the author should be directed back to the Post list

Given the author decides not to confirm the delete
When they the select the option to reject confirmation
Then the Post should NOT be removed from the system
And the author should be directed back to the Post details

________________________________________
# Testing Instructions for ticket #8

* In your terminal, from your current branch:
-git add and git commit whatever you're working on in your branch
-git checkout main
-git fetch --a or git fetch all
-(if files do not show, git pull origin)
-git checkout TSedit#8
-start the application with "start <application name>"

* Ensure that you have created the Tabloid database
* If needed:

-Use this to fill out tables and columns: 

[SQL data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-bit-o-honeys/blob/main/SQL/01_TabloidMVC_Create_DB.sql)

-Use this to seed the database: 

[SQL Seed Data](https://github.com/NewForce-Cohort-6/tabloidmvc-the-tootsie-rolls/blob/main/SQL/02_TabloidMVC_Seed_Data.sql)

* Launch the app from VisualStudio using the green outlined "play" button
* In the browser, if not logged in already, log in by selecting "login" and entering your email
* Once logged in, select the "Posts" tab from the navbar
* Once the posts tab is selected, you should see a list of the posts with options to view details, edit, or delete each post.
* Select the edit option. If the post was created by the logged in user, you should be taken to an editing screen where you can edit and save.
* Once save is clicked, you should be returned to the post list and your changes saved, which can be checked by viewing the edited posts details.

* If the post does not belong to the current user, a view will be shown informing the user that you an only edit your own posts with an option to return to the post list.

_____________
# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] My changes generate no new warnings or errors
- [x] I have added test instructions that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by sarahebrooks12** [comment]
Testing instructions look good --- get 2 testing approvals before merging 👼

**Review by shanedbutler — APPROVED** [review]
Tested working. Nice!

**Pull request merged**
